src/test/java/io/github/murdos/easyrandom/protobuf/Protobuf3MessageGenerationTest.java

```diff
@@ -57,15 +57,14 @@ void generatedValuesShouldBeValidAccordingToValidationConstraints() {
             assertThat(embeddedMessage.getStringField()).isNotBlank();
             assertThat(embeddedMessage.getEnumField()).isIn((Object[]) Proto3Enum.values());
         });
+        assertThat(protoInstance.getOneofFieldCase()).isNotEqualTo(Proto3Message.OneofFieldCase.ONEOFFIELD_NOT_SET);
     }
 
     @Test
     void shouldUseCollectionSizeRangeParameters() {
         EasyRandomParameters parameters = new EasyRandomParameters().collectionSizeRange(3, 3);
         EasyRandom easyRandom = new EasyRandom(parameters);
-
```

Comment:
Please keep theses lines, there are used to separate the 3 steps of the tests: Given, When, Then